bots/configure-bot-environment/README.md

```diff
@@ -1,57 +1,44 @@
-# Configuring bot environments
+# Configure bot environments
 
-Some configurations are needed by the SDK and are independent of bot type (i.e.
-Hubot, Botkit, Rasa etc).
+Regardless of bot type (e.g. Hubot, Botkit, Rasa, etc.), Rocket.Chat SDK requires you to make some configuration.
+This configuration can be stored in a `.env` file, for local development, for example.
+In production, they would need to be set on server startup.
 
-At the time of this writing only the Hubot adapter is production ready, but more
-are in development.
-
-In local development, the following can be set in an `.env` file. In production
-they would need to be set on server startup.
-
-`*` Required settings
+The following is the list of all the environmental variables you can use in your project. Variables marked with an asterisk (`*`) are mandatory.
```

Comment:
> The following is the list 

The following is a the list of all the environmental variables you can use in your project.
or 
This is a list of all the environmental variables you can use in your project.